src/sql/parts/notebook/notebook.component.ts

```diff
@@ -159,6 +170,11 @@ export class NotebookComponent extends AngularDisposable implements OnInit {
 		await model.requestModelLoad(this.notebookParams.isTrusted);
 		model.contentChanged((change) => this.handleContentChanged(change));
 		this._model = model;
+		if(this._actionBar)
```

Comment:
Please add this with the other taskbar items. There's no reason this shouldn't be shown at the start, but just be disabled until this is loaded.

Reply:
Changed the implementation. Action is added along with other task bar items. Shows as disabled by default until the model is loaded, then the button is displayed based on trusted mode.